assets/src/components/Treeview.js

```diff
@@ -51,13 +51,18 @@ export default class Treeview extends HTMLElement {
                         ? html`<div class="loading ${item.loadStatus === MapLayerLoadStatus.Loading ? 'spinner' : ''}"></div>`
                         : ''
                     }
-                    <input type="checkbox" class="${layerTreeGroupState.mutuallyExclusive ? 'rounded-checkbox' : ''}" id="node-${item.name}" .checked=${item.checked} @click=${() => item.checked = !item.checked} >
+                    <input type="checkbox"
+                        class="${layerTreeGroupState.mutuallyExclusive ? 'rounded-checkbox' : ''}"
+                        id="node-${item.name}"
+                        .checked=${item.checked}
+                        @click=${() => item.checked = !item.checked}
+                        @dblclick=${() => this._dblclicked(item)} >
                     <div class="node ${item.isFiltered ? 'filtered' : ''}">
                         ${item.type === 'layer'
                             ? html`<img class="legend" src="${item.icon}">`
                             : ''
                         }
-                        <label for="node-${item.name}">${item.layerConfig.title}</label>
+                        <label for="node-${item.name}" style="user-select: none;" @dblclick=${(e) => {e.preventDefault(); this._dblclicked(item);}} >${item.layerConfig.title}</label>
```

Comment:
Better put styles in CSS.

Reply:
@nboisteault have you an idea how to do so in CSS ? which file ? which css rule ?